src/sanitize-argument.ts

```diff
@@ -5,11 +5,12 @@ import Redis from './Redis';
 import urlParser from 'url';
 import logger from './Logger';
 import { config } from './config';
-import fs, { readFileSync } from 'fs';
+import fs from 'fs';
 import * as QueryStringParser from 'querystring';
-import { isValidEmail, ensureTrailingChar } from './util';
+import { isValidEmail, getCustomFlavorPath } from './util';
```

Comment:
Please move the `getCustomFlavorPath` code here and rename it to `sanitize_custon_flavour`, You can then remove `src/util/customFlavor.ts`,